### DISCUSSION TITLE
Redirect URLs

### DISCUSSION THREAD

**Discussion by lachlanglen** [near/wallet-selector#583] [General]
Hi,

It seems that there is no option to provide a `successUrl` or `fallbackUrl`/`errorUrl` when sending a transaction request (e.g. function call) - only when signing in.

Am I missing something? This is an essential feature for any apps that use redirect flows and need to provide custom URLs to handle the receiving end of redirects.

Thanks!

**Comment by kujtimprenkuSQA** [comment]
Hey, thank you for opening this discussion.

When sending transactions with browser wallets you can provide an optional param `callbackUrl`.

The other options mentioned above have no meaning at the moment because none of the wallets detect/accept these params, we could add them to our types but that would be just some dead code which would confuse developers even more.

I think wallets need to support first these params and then we can adjust to that implementation.
